README.md

```diff
@@ -420,6 +420,31 @@ $ friends add nickname "Grace Hopper" "Amazing Grace"
 Nickname added: "Grace Hopper (a.k.a. The Admiral a.k.a. Amazing Grace)"
 ```
 
+#### Add a default location
+
+You can set a default location in`friends`. Thereafter, whenever an activity is added and does not mention a location, the activity will be automatically 'tagged' with the default location.
```

Comment:
I think we might be able to tighten this language up a bit. What are your thoughts on something like:

When an activity includes the phrase `to _LOCATION_` (e.g., `Took a plane to _Paris_`), all future activities that have no explicit location will be associated with that location:

```bash
$ friends add activity Took a plane to Paris
Activity added: "2020-01-04: Took a plane to Paris"
Default location set: "Paris"
$ friends add activity Ate lunch at a charming café
Activity added: "2020-01-04: Ate lunch at a charming café"
$ friends add activity Left the city to go to _Chamonix_
Activity added: "2020-01-04: Left the city to go to Chamonix"
```

```bash
$ friends list activities --in Paris
2019-01-04: Ate lunch at a charming café
2019-01-04: Took a plane to Paris
```

Reply:
I like your description - and this feature deserves explicit examples. I'll put your change in 👍